core/permission/access.go

```diff
@@ -76,17 +76,19 @@ type ObjectType string
 
 // These values must match the values in the permission_object_type table.
 const (
-	Cloud      ObjectType = "cloud"
-	Controller ObjectType = "controller"
-	Model      ObjectType = "model"
-	Offer      ObjectType = "offer"
+	Cloud       ObjectType = "cloud"
+	Controller  ObjectType = "controller"
+	Model       ObjectType = "model"
+	Offer       ObjectType = "offer"
+	Application ObjectType = "application"
+	Unit        ObjectType = "unit"
```

Comment:
Is this change related to the secret?

Reply:
Yes, we need to record permissions for secret grants